Formula/pypy3.rb

```diff
@@ -11,23 +11,20 @@ class Pypy3 < Formula
     sha256 "1c13a907bd266e44d17dc6c5c39d59f4cf88e7ad89a67ad8b300a25481ec7bd0" => :el_capitan
   end
 
-  option "without-bootstrap", "Translate Pypy with system Python instead of " \
-                              "downloading a Pypy binary distribution to " \
-                              "perform the translation (adds 30-60 minutes " \
-                              "to build)"
+  option "without-pypy", "Translate Pypy with system Python instead of using " \
+                         "Homebrew's to perform the translation (adds 30-60 " \
+                         "minutes to build)" \
```

Comment:
what's with the trailing slash here?

Reply:
That would be a silly mistake. I’ll fix it later.